### PR TITLE
W-16710535-Update-parameters-table-CP

### PR DESCRIPTION
**Pull request by Crispy-Salesforce**
W-16710535-Update-parameters-table-CP

# Writer's Quality Checklist

Before merging your PR, did you:

- [ ] Run spell checker
- [ ] Run link checker to check for broken xrefs
- [ ] Check for orphan files
- [ ] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

### CODE REVIEW

File: modules/ROOT/pages/apikit-asyncapi-module-reference.adoc [hunk 1]
```diff
@@ -40,8 +40,8 @@ Default configuration
 | Name | Type | Description | Default Value | Required
 | Name | String | Name for this configuration. Connectors reference the configuration with this name. | None | Yes
 | Api Definition | String |  API definition. | None | Yes
-| Kafka Configs | Array of <<KafkaConfig>>. | None | No |  
-| Anypoint MQ Configs | Array of <<AnypointMqConfig>>. | None | No |  
+| Kafka Configs | Array of <<KafkaConfig>>. | Configuration settings for Kafka integration. | None | No  
+| Anypoint MQ Configs | Array of <<AnypointMqConfig>>. | Configuration settings for Anypoint MQ integration. | None | No
 | Name | String | Identifier of this element used to reference it in other components. | None | Yes
```
Comment (aby-philip-sf) [inline modules/ROOT/pages/apikit-asyncapi-module-reference.adoc:43]: For october we need to add two more protocols - solace, salesforce pubsub. I will share the reference doc in the slack channel.

File: modules/ROOT/pages/apikit-asyncapi-module-reference.adoc [hunk 2]
```diff
@@ -40,8 +40,8 @@ Default configuration
 | Name | Type | Description | Default Value | Required
 | Name | String | Name for this configuration. Connectors reference the configuration with this name. | None | Yes
 | Api Definition | String |  API definition. | None | Yes
-| Kafka Configs | Array of <<KafkaConfig>>. | None | No |  
-| Anypoint MQ Configs | Array of <<AnypointMqConfig>>. | None | No |  
+| Kafka Configs | Array of <<KafkaConfig>>. | Configuration settings for Kafka integration. | None | No
```
Comment (aby-philip-sf) [inline modules/ROOT/pages/apikit-asyncapi-module-reference.adoc:41]: FYI - this is a list of kafka server configurations. One for each kafka server listed in the specification that was scaffolded to create the app.

Comment (Crispy-Salesforce) [reply modules/ROOT/pages/apikit-asyncapi-module-reference.adoc:41]: OK! Thanks!
Adding these comments to the following PR: https://github.com/mulesoft/docs-apikit/pull/201/files

File: modules/ROOT/pages/apikit-asyncapi-module-reference.adoc (hunk 1 above)
Comment (Crispy-Salesforce) [reply modules/ROOT/pages/apikit-asyncapi-module-reference.adoc:43]: OK! Thanks!
Adding these comments to the following PR: https://github.com/mulesoft/docs-apikit/pull/201/files

**Comment by Crispy-Salesforce** [comment]
The changes and comments in this PR were included in the following PR:
https://github.com/mulesoft/docs-apikit/pull/201/files
Closing PR.

**Pull request closed without merging**